### DISCUSSION TITLE
[Raster Math tool] How to work with NoData

### DISCUSSION THREAD

**Discussion by vprint** [EnMAP-Box/enmap-box#203] [Q&A] [answered]
I have 2 reclassified raster: one for dry area (class 4 to 6) and one for wetland area(class 1 to 3)
If I sum them up to obtain a global classification I get a strange output because NoData values (-9999) are not taken into account:
![RasterCalculator](https://user-images.githubusercontent.com/42189444/196009885-2b63386a-2ad8-47b5-b1a3-cc5948b866bb.gif)

Is it possible to assign 0 to NoDataValues in the raster calculator tool?

Here are my datas

[NoDataRasterMathBug.zip](https://github.com/EnMAP-Box/enmap-box/files/9793194/NoDataRasterMathBug.zip)

**Comment by janzandr** [comment, marked as answer]
Does that look right?
![image](https://user-images.githubusercontent.com/45030715/196010388-3769b54f-aaf8-4e24-a632-e026aaa71e02.png)

`np.maximum(0, Land_Reclass) + np.maximum(0, WetlandArea_Reclassified)`

**Reply by vprint** [reply]
This works fine
